src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  path: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ path, text }) => {
+  return (
+    <NavLink
+      to={path}
+      className={({ isActive }) => classNames('nav__link', {
+        'nav__link--active': isActive,
+      })}
```

Comment:
```suggestion
      className={classNames('nav__link', {
        'nav__link--active': isActive,
      })}
```

Reply:
I can't remove isActive, because I don't store it in the state. I use this NavLink feature
![image](https://github.com/mate-academy/react_phone-catalog/assets/98056446/c73dce9b-5634-4d17-a37a-c945b8509438)